docs/glossary.md

```diff
@@ -5,35 +5,84 @@ sidebar_label: Glossary
+| System                        | Group of components. A system can be horizonal or vertical. Horizontal is organisationally cross-cutting, for example 'Azure'. Vertical is a grouping for an organisational vertical, eg an application.                                        |
+| System Owner                  | Implementors of controls.                                                                                                                                                                                                                       |
+| System Security Plan          | A description of the control implementation of an information system.                                                                                                                                                                           |
+| SMF24                         | See 'Senior Management Function 24'                                                                                                                                                                                                             |
+| Variance                      | Deviations or exceptions from established security standards, policies, or procedures.                                                                                                                                                          |
+
+## Internal Architecture Glossary
+
+These terms are when we refer to the internal architecture that composes Compliance Framework. We should stick to it to not make our lives harder.
+
+| Name                          | Definition                                                                                                                                                                                                                                      |
+|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Assessment Runtime            | See [Assessment Runtime](/docs/architecture/assessment-runtime.md)                                                                                                                                                                              |
+| Core Services                 | See [Argus Core](/docs/architecture/core/index.md)                                                                                                                                                                                              |
+| Dashboard UI                  | See [...]                                                                                                                                                                                                                                       |
```

Comment:
We can change this to "Portal" as the word has a broader scope aligning with the app's purpose.